src/ast/passes/codegen_llvm.h

```diff
@@ -280,19 +274,74 @@ class CodegenLLVM : public Visitor {
 
   std::map<std::string, AllocaInst *> variables_;
 
-  // NB: ensure all IDs are saved/restored in create_reset_ids()
-  int printf_id_ = 0;
-  int mapped_printf_id_ = 0;
-  int time_id_ = 0;
-  int cat_id_ = 0;
-  int strftime_id_ = 0;
-  uint64_t join_id_ = 0;
-  int system_id_ = 0;
-  int non_map_print_id_ = 0;
-  uint64_t watchpoint_id_ = 0;
-  int cgroup_path_id_ = 0;
-  int skb_output_id_ = 0;
-  int str_id_ = 0;
+#define ADD_ASYNC_ID(ID)                                                       \
+  int ID = 0;                                                                  \
```

Comment:
Can we mangle this variable name so that direct access is more obvious during code review? Maybe mangle it like `baddirectaccess_ # ID`?